### PR TITLE
Changes PKM examine

### PR DESCRIPTION
**Pull request by peoplearestrange**
## About The Pull Request
Changes examine to make it obvious to use a crowbar to remove modules

## Why It's Good For The Game
Its not obvious that you need to use a crowbar to remove modules when other "modules" such as bayonets and flashlights are removed with a screwdriver

## Changelog
:cl:
tweak: PKM examine tells you to use a crowbar for modules
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 1]
```diff
@@ -137,7 +137,7 @@
 /obj/item/gun/energy/kinetic_accelerator/proc/reload()
 	cell.give(cell.maxcharge)
 	if(!suppressed)
-		playsound(src.loc, 'sound/weapons/kenetic_reload.ogg', 60, 1)
+		playsound(src.loc, 'sound/s/kenetic_reload.ogg', 60, 1)
```
Comment (peoplearestrange) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: WHOOPS!

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 2]
```diff
@@ -33,10 +33,10 @@
 			var/obj/item/borg/upgrade/modkit/M = A
 			. += "<span class='notice'>There is \a [M] installed, using <b>[M.cost]%</b> capacity.</span>"
 
-/obj/item/gun/energy/kinetic_accelerator/crowbar_act(mob/living/user, obj/item/I)
+/obj/item/gun/energy/kinetic_accelerator/screwdriver_act(mob/living/user, obj/item/I)
 	. = TRUE
 	if(modkits.len)
-		to_chat(user, "<span class='notice'>You pry the modifications out.</span>")
+		to_chat(user, "<span class='notice'>You unscrew the modifications out.</span>")
```
Comment (SpaceManiac) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: Weird nitpick, but "You unscrew the modifications out." reads somewhat awkwardly. I don't think "to unscrew out" is common usage. Maybe "You unfasten the modifications." or something along those lines?

**Review by SpaceManiac — CHANGES_REQUESTED** [review]

**Comment by Dawson1917** [comment]
It's easier because miners spawn with a crowbar

**Comment by YPOQ** [comment]
I think you'll need to add a parent call to `screwdriver_act()` so players can remove flashlights

**Comment by CRITAWAKETS** [comment]
Maybe replace the crowbar that is in the box miners get with a screwdriver?

**Comment by kriskog** [comment]
Screwdriver is already used to remove flashlight and bayonet, make sure that behaviour is preserved. Otherwise, having a crowbar in my box is much more useful than a screwdriver.

**Comment by HomocidlePancakes** [comment]
1)Miners start with a crowbar in case of a power outage. 
2)KA mods are removed by a crowbar because miners start with them.
3)We already have an interaction with KAs designated for screwdriver

**Comment by peoplearestrange** [comment]
Ah crud you're completely right. Hmm how about adding a line when they examine to show that they can use a crowbar to remove modules then?

**Review by peoplearestrange — COMMENTED** [review]
Just changes the examine to make it more obvious you use a crowbar for modules

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm [hunk 3]
```diff
@@ -28,7 +28,7 @@
 /obj/item/gun/energy/kinetic_accelerator/examine(mob/user)
 	. = ..()
 	if(max_mod_capacity)
-		. += "<b>[get_remaining_mod_capacity()]%</b> mod capacity remaining."
+		. += "<b>[get_remaining_mod_capacity()]%</b> mod capacity remaining. You can use a crowbar to remove modules"
```
Comment (SpaceManiac) [inline code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: Needs `.` at end

Comment (ShizCalev) [reply code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: follow the formatting.
new line, then
`<span class='info'>You can use a <b>crowbar </b> to remove modules.</span>`

**Comment by SpaceManiac** [comment]
Waiting on reply

File: code/modules/projectiles/guns/energy/kinetic_accelerator.dm (hunk 2 above)
Comment (optimumtact) [reply code/modules/projectiles/guns/energy/kinetic_accelerator.dm]: unfasten is the right word

**Comment by SpaceManiac** [comment]
@ShizCalev felt bad closing a trivial one-liner for being stale, just fixed it myself, please review

**Pull request merged**